pyiron_base/jobs/job/extension/server/generic.py

```diff
@@ -184,15 +184,19 @@ def queue(self, new_scheduler):
                 )
                 if cores != self.cores:
                     self._cores = cores
-                    state.logger.debug("Updated the number of cores to: %i", cores)
+                    state.logger.warning(
+                        "Updated the number of cores to: %i", RuntimeWarning
```

Comment:
```suggestion
                        f"Updated the number of cores to: {corrs}", RuntimeWarning
```
Obviously, these things are not tested... will add tests and fix these errors...